src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -164,11 +170,11 @@
                      id="unmanaged"
                      type="checkbox"
                      formControlName="unmanaged">
-              <label class="custom-control-label"
+              <label class="custom-control-label m-0"
                      for="unmanaged"
                      i18n>Unmanaged</label>
-              <cd-helper i18n>If set to true, the orchestrator will not start nor stop any daemon associated with this service.
-                 Placement and all other properties will be ignored.</cd-helper>
+              <cd-help-text i18n>If set to true, the orchestrator will not start nor stop any daemon associated with this service.
+                 Placement and all other properties will be ignored.</cd-help-text>
```

Comment:
I would say, do we need the unmanaged flag in general usecase. Why sould someone not like cephadm to manage the service in production env?

Reply:
I think we should keep it anyway as its present for all other services. cc @nizamial09 any thoughts ??

Reply:
unmanaged is useful for administrator to do troubleshooting. Once the set the flag, cephadm will stop doing any automation its being doing on the daemon which will help for a more admin controlled diagnosis which IMO is good to have. So I think it should be there. Also its part of the service_spec (a required one too), we might need to always send a false value to the backend which is not ideal as well. So I'd say let it be there.